stl/inc/xcharconv_ryu.h

```diff
@@ -1152,7 +1152,7 @@ _NODISCARD inline __floating_decimal_32 __f2d(const uint32_t __ieeeMantissa, con
       __lastRemovedDigit = 4;
     }
     // We need to take __vr + 1 if __vr is outside bounds or we need to round up.
-    __output = __vr + ((__vr == __vm && (!__acceptBounds || !__vmIsTrailingZeros)) || __lastRemovedDigit >= 5);
+     = __vr + ((__vr == __vm && (!__acceptBounds || !__vmIsTrailingZeros)) || __lastRemovedDigit >= 5);

```

Comment:
I think here should be `_Output`

Reply:
Yeah thanks, seems I accidentally removed it.